src/github.com/couchbase/sync_gateway/base/bucket.go

```diff
@@ -55,15 +55,56 @@ type BucketSpec struct {
 
 // These are used by CBGT to determine the sharding factor and other properties
 type FeedParams struct {
-	MaxPartitionsPerPIndex int `json:"max_partitions_per_pindex"`
-	NumReplicas            int `json:"num_replicas"`
+	NumShards   int `json:"num_shards"`
+	NumReplicas int `json:"num_replicas"`
```

Comment:
My only take is that I like coding in python where I never have to think about this stuff!

What should I change these to?  uint16?